src/main/java/com/graphhopper/converter/api/NominatimEntry.java

```diff
@@ -27,14 +28,39 @@ public NominatimEntry(long osmId, String type, double lat, double lon, String di
         this.displayName = displayName;
 
         this.address = new Address();
-        address.setCountry(country);
-        address.setCity(city);
+        address.country = country;
+        address.city = city;
     }
 
     public NominatimEntry() {
         this.address = new Address();
     }
 
+    public boolean isStreet() {
+        if ("highway".equals(this.classString)) {
+            return true;
+        }
+        return false;
```

Comment:
I am not sure, which is better. Sometimes I think the longer version is better, sometimes I think the shorter is better. Here I thought that we might add further cases to decide if an entry is a street, therefore I used the long version. I use the short version for now.